### DISCUSSION TITLE
prefix just doesent't change

### DISCUSSION THREAD

**Discussion by Intergreece** [jagrosh/MusicBot#1421] [Troubleshooting] [answered]
i've downloaded jar file, placed it in custom folder in Documents, launched it once to get the config.txt, modified it:
![изображение_2023-08-13_125216149](https://github.com/jagrosh/MusicBot/assets/118989032/b7fb4836-c926-4292-95fc-719909800b2a)
however the bot only replies to @mentoin as if i left the prefix = ""
![Без имени](https://github.com/jagrosh/MusicBot/assets/118989032/1fc8f622-7fa4-407c-9b1c-fa497e8817f0)
(if you think the trouble is in command it's listed in file moreover you can see the bot's response in the 2nd pic)
![image](https://github.com/jagrosh/MusicBot/assets/118989032/b5b29608-3d11-4875-921f-a24d45b141c9)

again, the bot sends help info and doing commands, just the prefix that doesen't work and i really don't want to use the bot by @mention
Thanks for help!

**Comment by MichailiK** [comment, marked as answer]
You need to go to your bot settings on the Discord Developer dashboard, and make sure you have "Message Content Intent" ticked.

![image](https://github.com/jagrosh/MusicBot/assets/39029839/56f5cc78-bbfc-4f01-9f66-2c852381b7e5)

**Reply by Intergreece** [reply]
thank you!!!
